File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
//declare height variable
    int height;
//get value for height between 1-8
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);
//print # in shape of pyramid
    for (int i = 0; i < height; i++)
    {
//printing spaces
        for (int k = height - 1 - i; k > 0; k = k - 1)
        {
            printf(" ");
        }
//printing #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int z = 0; z <= i; z++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# get height
height = get_int("Height: \n")
while height < 1 or height > 8:
    height = get_int("Input valid height\n")

# print #'s out
for i in range(height):
    for x in range(height - 1 - i):
        print(" ", end="")
    for y in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for x in range(i + 1):
        print("#", end="")
    print()